satpy/readers/viirs_l2.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2022-2023 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Interface to VIIRS L2 files."""
+
+from datetime import datetime
+
+from satpy.readers.netcdf_utils import NetCDF4FileHandler
+
+
+class VIIRSCloudMaskFileHandler(NetCDF4FileHandler):
+    """VIIRS L2 Cloud Mask reader."""
+
+    def __init__(self, filename, filename_info, filetype_info):
+        """Initialize the file handler."""
+        super().__init__(filename, filename_info, filetype_info, cache_handle=True)
+
+    def _parse_datetime(self, datestr):
+        """Parse datetime."""
+        return datetime.strptime(datestr, "%Y-%m-%dT%H:%M:%SZ")
+
+    @property
+    def start_orbit_number(self):
+        """Get start orbit number."""
+        return int(self['/attr/start_orbit_number'])
+
+    @property
+    def end_orbit_number(self):
+        """Get end orbit number."""
+        return int(self['/attr/end_orbit_number'])
+
+    @property
+    def platform_name(self):
+        """Get platform name."""
+        try:
+            res = self.filename_info['platform_shortname']
+        except KeyError:
+            res = 'Unknown'
```

Comment:
Is this needed or should the reader just crash if the platform isn't known? also according to codecov, this isn't tested.

Reply:
That's another leftover from the reader I based this on. Could've been `viirs_l1b.py` where this exact bit is also. Removed the try/except in e8fcaaf9a